src/Rx.KitchenSink.ts

```diff
@@ -13,6 +13,8 @@ interface KitchenSinkOperators<T> extends CoreOperators<T> {
   min?: <T, R>(comparer?: (x: R, y: T) => R) => Observable<R>;
   timeInterval?: <T>(scheduler?: IScheduler) => Observable<T>;
   mergeScan?: <T, R>(project: (acc: R, x: T) => Observable<R>, seed: R) => Observable<R>;
+  switchFirst?: <R>() => Observable<R>
```

Comment:
Indeed. I fixed it.

The `switch` operator is also wrong, I'll do a small PR to fix that. Thanks !